bitshares-ubuntu-vm/azuredeploy.json

```diff
@@ -0,0 +1,209 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Username for the Virtual Machine."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the Virtual Machine."
+      }
+    },
+    "dnsLabelPrefix": {
+       "type": "string",
+       "metadata": {
+          "description": "Hostname assigned to the Public IP, also used as the VM Name. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+       }
+    },
+    "installMethod": {
+      "type": "string",
+      "defaultValue": "From_PPA",
+      "allowedValues": [
+        "From_PPA",
+        "From_Source"
+      ],
+      "metadata": {
+        "description": "Method to install bits. From_PPA: unofficial community bianaries.  From_Source: official BitShares repo on GitHub."
+      }
+    },    
+    "vmSize": {
+      "type": "string",
+      "defaultValue": "Standard_A2",
+      "allowedValues": [
+        "Standard_A1",
+        "Standard_A2",
+        "Standard_A3",
+        "Standard_D1_V2",
+        "Standard_D2_V2",
+        "Standard_D3_V2"
+      ],
+      "metadata": {
+        "description": "Size of VM"
+      }
+    }
+  },
+  "variables": {
+    "imagePublisher": "Canonical",
+    "imageOffer": "UbuntuServer",
+    "OSDiskName": "BitSharesOSDisk",
+    "imageSKU": "15.10",
+    "nicName": "BitSharesNic",
+    "addressPrefix": "10.0.0.0/16",
+    "subnetName": "BitSharesSubnet",
+    "subnetPrefix": "10.0.0.0/24",
+    "storageAccountType": "Standard_LRS",
+    "storageAccountName": "[replace(replace(tolower(concat('bitsharessa', uniquestring(resourceGroup().id))), '-',''),'.','')]",
```

Comment:
Use the unique string first, it helps with spreading storage accounts from this template on different clusters. Also you don't need to replace and toLower output of uniquestring. It already is compatible with storage account names.